File: Examples/CSharp/Loading-and-Saving/NodeTypeEnumeration.cs

using System.IO;
using Aspose.Note;
using System;

namespace CSharp.Loading_Saving
{
    public class NodeTypeEnumeration
    {
        public static void Run()
        {
            // ExStart:NodeTypeEnumeration
            // The path to the documents directory.
            Document doc = new Document();

            // Returns NodeType.Document
            NodeType type = doc.NodeType;
            // ExEnd:NodeTypeEnumeration
           
        }
    }
}

File: Examples/VisualBasic/Loading-and-Saving/NodeTypeEnumeration.vb
Imports System.IO
Imports Aspose.Note

Namespace Loading_Saving
    Public Class NodeTypeEnumeration
        Public Shared Sub Run()
            ' ExStart:NodeTypeEnumeration
            ' The path to the documents directory.
            Dim doc As New Document()

            ' Returns NodeType.Document
            Dim type As NodeType = doc.NodeType
            ' ExEnd:NodeTypeEnumeration
        End Sub
    End Class
End Namespace